realm/realm-annotations-processor/src/test/resources/io/realm/BooleansRealmProxy.java

```diff
@@ -15,7 +13,9 @@
 import io.realm.internal.Row;
 import io.realm.internal.SharedRealm;
 import io.realm.internal.Table;
+import io.realm.internal.UncheckedRow;
```

Comment:
Got it.  Please see comment above.  I suggest @SuppressWarnings("all") in the header of these classes.  Just suggesting.  It seems like a reasonable idea to me but I bow before the opinion of the team.